### PR TITLE
Add BER MetaOCaml N111

### PR DESCRIPTION
**Pull request by yallop**
See: https://sympa.inria.fr/sympa/arc/caml-list/2020-10/msg00007.html

### CODE REVIEW

**Comment by mseri** [comment]
Thanks

File: packages/ocaml-variants/ocaml-variants.4.11.1+BER/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "base-threads" {post}
+  "base-metaocaml-ocamlfind" {post}
+]
+conflict-class: "ocaml-core-compiler"
+flags: compiler
+setenv: CAML_LD_LIBRARY_PATH = "%{lib}%/stublibs"
+build: [
+  ["./configure" "-prefix" "%{prefix}%"]
+    {os != "openbsd" & os != "freebsd" & os != "macos"}
+  ["./configure" "-prefix" "%{prefix}%" "-cc" "cc" "-aspp" "cc -c"]
```
Comment (mseri) [inline packages/ocaml-variants/ocaml-variants.4.11.1+BER/opam:17]: @yallop: see https://github.com/ocaml/opam-repository/issues/17385
Should we update this file to use 
```
["./configure" "-prefix" "%{prefix}%" "CC=cc" "-aspp" "cc -c"]
```
instead?

Comment (yallop) [reply packages/ocaml-variants/ocaml-variants.4.11.1+BER/opam:17]: @meri: that sounds good to me.  See https://github.com/ocaml/opam-repository/pull/17386

Comment (mseri) [reply packages/ocaml-variants/ocaml-variants.4.11.1+BER/opam:17]: Thanks!

**Pull request merged**